src/app/Tools/Settings/BookmarkEditor.tsx

```
@@ -0,0 +1,1196 @@
+            return;
+        }
+
+        setEditedBookmark(prev => ({
+            ...prev,
+            Keywords: [...(prev.Keywords || []), newKeyword.trim()]
+        }));
+
+        setNewKeyword("");
+        setErrors(prev => {
+            const newErrors = { ...prev };
+            delete newErrors.keyword;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)